pkg/cableengine/cableengine_test.go

```diff
@@ -177,40 +184,62 @@ var _ = Describe("Cable Engine", func() {
 				})
 			}
 
+			// Run the tests for both IPv4 and IPv6
 			Context("with a different cable name", func() {
 				BeforeEach(func() {
 					newEndpoint.Spec.CableName = "new cable"
 				})
 
-				testTimestamps()
+				for _, family := range []k8snet.IPFamily{k8snet.IPv4, k8snet.IPv6} {
+					Context(fmt.Sprintf("using IP family %v", family), func() {
```

Comment:
So it read "_using IPv4_" instead of "_using IP family 4_".

```suggestion
					Context(fmt.Sprintf("using IPv%v", family), func() {
```